x-pack/legacy/plugins/watcher/public/services/action_defaults/actions/email_action.js

```diff
@@ -8,12 +8,11 @@ import { merge } from 'lodash';
 import { ACTION_TYPES, WATCH_TYPES } from 'plugins/watcher/../common/constants';
 import { ActionDefaultsRegistryProvider } from '../registry';
 
-const DEFAULT_ADMIN_EMAIL_CONFIG_KEY = 'xPack:defaultAdminEmail';
 const actionType = ACTION_TYPES.EMAIL;
 
-function getActionDefaults(config) {
+function getActionDefaults() {
   return {
-    to: config.get(DEFAULT_ADMIN_EMAIL_CONFIG_KEY)
+    to: ''
```

Comment:
cc @elastic/elastic-ui team - this might be a feature regression - I think you are in the process of refactoring this so heads up about this change

Reply:
Please don't ping that team, it's not just Kibana

Reply:
@bevacqua Thanks for the call out. I had the wrong alias. Updated

Reply:
@chrisronline thanks for the heads up! i'll update the refactored watcher code (https://github.com/elastic/kibana/pull/35301).

Reply:
Fixed in this commit: https://github.com/elastic/kibana/pull/35301/commits/ccecb29cf1511cab135ebe672391186331f69307

Reply:
hey @alisonelizabeth as a heads up I (added and then) dropped the 7.x label from this, ref https://github.com/elastic/kibana/pull/33603#issuecomment-506486054.  only 8.0 as the urgency isn't too high